test.py

```diff
@@ -0,0 +1,14 @@
+def add_apples(func):
+    def get_fruits():
```

Comment:
@vendethiel got some ignored files commited. This is stale pull request, review a [new one](https://github.com/adambard/learnxinyminutes-docs/pull/2394), please.